applications/imaging_ai_segmentator/README.md

```diff
@@ -1,198 +1,175 @@
+## Build and Run Instructions
 
 ### Quick Start Using Holohub Container
 
-This is the simplest and fastest way to see the application in action running as a container. The input DICOM files must first be downloaded and saved in the folder `$PWD/data/imaging_ai_segmentator/dicom`, whereas the PyTorch model is automatically downloaded when container image is built.
-
-Use the following to build and run the application:
+This is the simplest and fastest way to run the application:
 
 ```bash
-mkdir -p output
-rm -rf output/*
-./dev_container build_and_run imaging_ai_segmentator --container_args "-v $PWD/output:/var/holoscan/output -v $PWD/data/imaging_ai_segmentator/dicom:/var/holoscan/input"
+./holohub run imaging_ai_segmentator
 ```
 
-Once the command completes, please check the output folder for the results, e.g.
-```
+**_Note:_** It takes quite a few minutes when this command is run the first time. This command pulls the latest Holoscan SDK docker image, create Holohub docker image and set up requirement for this application, run the container, and finally build and run the application.
+
+The output will be available in the `"<LOCAL_HOLOHUB_PATH>/build/imaging_ai_segmentator/output"` directory, where `<LOCAL_HOLOHUB_PATH>` refers to where you have cloned your Holohub repository and running the `./holohub` command.
+
+```console
 output
 ├── 1.2.826.0.1.3680043.10.511.3.57591117750107235783166330094310669.dcm
 └── saved_images_folder
     └── 1.3.6.1.4.1.14519.5.2.1.7085.2626
         ├── 1.3.6.1.4.1.14519.5.2.1.7085.2626.nii
         └── 1.3.6.1.4.1.14519.5.2.1.7085.2626_seg.nii
-
-2 directories, 3 files
 ```
 
-**_Note_**
-It takes quite a few minutes when this command is run the first time.
-
-### Run the Application in Dev Environment
-
-It is strongly recommended a Python virtual environment is used for running the application in dev environment.
-
-This application only has Python implementation depending on a set of Python packages from [Pypi](https://pypi.org), however, a `build_and_install` step is needed to automate organizing Python code and downloading the model.
+You can modify them by setting the right env variables and mount the right volumes, for instance:
 
-
-Set up the Holohub environment, if not already done
 ```bash
-./run setup
+./dev_container build_and_run imaging_ai_segmentator --container_args "-v /local/output:/my_output -e HOLOSCAN_OUTPUT_PATH=/my_output"
 ```
 
-Set the environment variables for the application
+By default, the application uses the following directories for input data, model files, and output results:
+
 ```bash
-source applications/imaging_ai_segmentator/env_settings.sh
+HOLOSCAN_INPUT_PATH=<LOCAL_HOLOHUB_PATH>/data/imaging_ai_segmentator/dicom
+HOLOSCAN_MODEL_PATH=<LOCAL_HOLOHUB_PATH>/data/imaging_ai_segmentator/models
+HOLOSCAN_OUTPUT_PATH=<LOCAL_HOLOHUB_PATH>/build/imaging_ai_segmentator/output
```

Comment:
These env var setting command serves no purpose here in this updated README
- they are only needed when a user runs the application in local env (outside of the dev_container), and Section 1.B below overrides them by sourcing the original env_setting.sh which automatically and correctly sets the `LOCAL_HOLOHUB_PATH`, i.e. `GIT_ROOT=$(readlink -f ./$(git rev-parse --show-cdup))`
- In (app specific) dev container, these env vars are set by the dockerfile, to different values which are container vol mapped.

Reply:
Will remove this too. Thanks